rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/mute/standalone_mute_runner.kt

```diff
@@ -1,5 +1,9 @@
+@file:JvmName("StandaloneMuteRunner")
```

Comment:
This is the only change that is expected in this PR, given the title

Reply:
:-)